packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement, useContext } from 'react';
+import { Html } from '@react-three/drei';
+
+import { ISceneNodeInternal } from '../../../store';
+import { sceneComposerIdContext } from '../../../common/sceneComposerIdContext';
+import { IDataOverlayComponentInternal } from '../../../store/internalInterfaces';
+
+import { DataOverlayContainer } from './DataOverlayContainer';
+
+export interface DataOverlayComponentProps {
+  node: ISceneNodeInternal;
+  component: IDataOverlayComponentInternal;
+}
+
+export function DataOverlayComponent({ node, component }: DataOverlayComponentProps): ReactElement {
```

Comment:
Why are you using the traditional `function` expression rather than implementing arrow functions? Are there plans to use scope as part of how these functions execute in the future? As of now, there are not any usages that would call for traditional functions so I'd encourage changing these to use arrows in order to clearly delineate the limitations.

Reply:
will update. I was doing some other testing and didn't change it back